src/snake/main.rs

```diff
@@ -281,6 +286,30 @@ impl<R: Read, W: Write> Game<R, W> {
         }
     }
 
+    fn game_start(&mut self) {
+        self.stdout.goto(0, 0).unwrap();
+        self.stdout.write_fmt(format_args!("╔══════════════════════════════════╗\n\r\
```

Comment:
Why formatting?

Also, this should be conditional, since Redox term is ASCII-only.